code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1165,19 +1165,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			H.satiety = -MAX_SATIETY
 		else if(H.satiety < 0)
 			H.satiety++
-			if(prob(round(-H.satiety/40)))
+			if(DT_PROB(round(-H.satiety/80), delta_time))
```

Comment:
Fix probability, might be difficult here again

Reply:
Corrected. The maximum deviation is now ~0.07%